taskcluster/scripts/pyenv-setup

```diff
@@ -3,6 +3,8 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+set -e
```

Comment:
👍 I see we don't have any secrets in this script. We can probably add for extra debugging too:

```suggestion
set -ex
```